.devcontainer/Dockerfile

```diff
@@ -24,24 +25,23 @@ COPY . .
 # install runtime from DIST if there is one
 
 # install runtime from DIST if there is one
-RUN mkdir -p /root/.local && \
+RUN mkdir -vp /root/.local && \
     if [ -d dist ] ; then \
     touch requirements.txt && \
-    pip install --user -r requirements.txt dist/*.whl ; \
+    pip install --no-cache --user -r requirements.txt dist/*.whl ; \
```

Comment:
Thanks for this tip - not something I have used before in a python container but makes good sense. 

It will make no difference in this case. The dist folder is not present during developer target build and this stage is dropped for the runtime build.

But lets keep this as a best practice reminder.